h2o-core/src/main/java/water/api/schemas3/ModelParametersSchemaV3.java

```diff
@@ -288,11 +290,17 @@ public static AutoBuffer writeParametersJSON(AutoBuffer ab, ModelParametersSchem
     }
 
     compute_transitive_closure_of_is_mutually_exclusive(metadata);
-
+    if (additionalParameters != null) {
+      metadata = (ModelParameterSchemaV3[]) ArrayUtils.addAll(metadata, additionalParameters);
```

Comment:
water.util.ArrayUtils#append(T[] a, T... b) handles the null check for you

Reply:
Thank you. Used our own ArrayUtils instead.